flow/activities/flowable_core.go

```diff
@@ -528,3 +535,19 @@ func replicateXminPartition[TRead any, TWrite any, TSync connectors.QRepSyncConn
 
 	return currentSnapshotXmin, nil
 }
+
+func (a *FlowableActivity) getPeerNameForMirror(ctx context.Context, flowName string, peerType PeerType) (string, error) {
```

Comment:
seems unnecessary, retrieve the config at the handler level and pass the names in from that

Reply:
`DropFlowWorkflow` takes in `ShutdownRequest` as input. This would require a new proto if you want DropFlowWorkflow to take flowName, sourceName and destName instead of just flowName

Reply:
give DropFlow FlowConnectionConfigs and pass the activities the names as needed?

Reply:
I prefer minimal api

Reply:
not sure how fetching the entire flow config is less unnecessary than just fetching the names. closing this for now